pkgs/development/libraries/protobuf/27.nix

```diff
@@ -0,0 +1,6 @@
+{ callPackage, ... } @ args:
+
+callPackage ./generic.nix ({
+  version = "27.0";
+  hash = "sha256-QVgDGyNTCSYU/rXTMcHUefUGVjwkkjnpq0Kq+eXK/bo=";
+} // args)
```

Comment:
Side note, I think it's kind of unfortunate that we use this "nested callPackage" pattern:

```
nix-repl> lib.functionArgs protobuf_26.override
{ callPackage = false; }
```